### DISCUSSION TITLE
How can i get this working from client-side?

### DISCUSSION THREAD

**Discussion by sjonniesensei** [karelkryda/universal-speedtest#30] [General]
The server side is working perfectly, but how can I get this working client-side through the browser?

Possible to add a CDN version of the universal-speedtest library that can be included in the HTML file as a <script> tag?

**Comment by karelkryda** [comment]
Hi,
I wonder if it is even possible for a library to function like this, at least given its dependencies.
I can try to see if I would be able to package a variant for the website. If so, I could try to do it before the release of the production version `3.0.0`, but I don't want to promise anything, I still have to finish the priority tasks first.

**Reply by sjonniesensei** [reply]
Hi @karelkryda ,

You trying is all we need. Thanks for the effort and I hope it will be possible.
I hope you have a great Wednesday.

**Reply by WebDevDatasur** [reply]
Hi,

Did you get time to check this out?

**Reply by karelkryda** [reply]
Hi,
not yet to be honest. I was sick and didn't have much time to devote to it. Before that, I managed to release another RC version, but in it I was more about fixing things and adding new functionalities.

There are a few more things planned and I have this on the list before the release of version `3.0.0`.

Thanks for your understanding

**Reply by sjonniesensei** [reply]
Alright, Thanks for the update.
